### PR TITLE
Add formula for prettyping

### PR DESCRIPTION
**Pull request by denilsonsa**
As suggested by @mikemcquaid on https://github.com/Homebrew/homebrew/pull/23354

Note, however, I wrote this blindly, because I don't have a Mac system and I cannot test it.

### CODE REVIEW

File: Library/Formula/prettyping.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require 'formula'
```
Comment (mikemcquaid) [inline Library/Formula/prettyping.rb]: don't need this line any more.

File: Library/Formula/prettyping.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+class Prettyping < ScriptFileFormula
+  homepage 'http://denilsonsa.github.io/prettyping/'
+  url 'https://raw.githubusercontent.com/denilsonsa/prettyping/v1.0.0/prettyping'
+  sha1 '5e0296f9f28834d0985bb9f3452e777b35291350'
+  version '1.0.0'
+  def install
+    bin.install 'prettyping'
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/prettyping.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/prettyping.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'formula'
+
+class Prettyping < ScriptFileFormula
+  homepage 'http://denilsonsa.github.io/prettyping/'
+  url 'https://raw.githubusercontent.com/denilsonsa/prettyping/v1.0.0/prettyping'
+  sha1 '5e0296f9f28834d0985bb9f3452e777b35291350'
+  version '1.0.0'
+  def install
```
Comment (mikemcquaid) [inline Library/Formula/prettyping.rb]: newline above this.

File: Library/Formula/prettyping.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+require 'formula'
+
+class Prettyping < ScriptFileFormula
+  homepage 'http://denilsonsa.github.io/prettyping/'
+  url 'https://raw.githubusercontent.com/denilsonsa/prettyping/v1.0.0/prettyping'
```
Comment (mikemcquaid) [inline Library/Formula/prettyping.rb]: Use https://github.com/denilsonsa/prettyping/archive/v1.0.0.tar.gz

File: Library/Formula/prettyping.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  sha1 '5e0296f9f28834d0985bb9f3452e777b35291350'
```
Comment (mikemcquaid) [inline Library/Formula/prettyping.rb]: Use `sha256`, thanks.

File: Library/Formula/prettyping.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  version '1.0.0'
```
Comment (mikemcquaid) [inline Library/Formula/prettyping.rb]: This probably isn't needed.

File: Library/Formula/prettyping.rb [hunk 7]
```diff
@@ -0,0 +1,11 @@
+require 'formula'
+
+class Prettyping < ScriptFileFormula
+  homepage 'http://denilsonsa.github.io/prettyping/'
```
Comment (mikemcquaid) [inline Library/Formula/prettyping.rb]: Use double-quotes everywhere.

File: Library/Formula/prettyping.rb [hunk 8]
```diff
@@ -0,0 +1,11 @@
+require 'formula'
+
+class Prettyping < ScriptFileFormula
```
Comment (xu-cheng) [inline Library/Formula/prettyping.rb]: `class Prettyping < Formula`

File: Library/Formula/prettyping.rb (hunk 6 above)
Comment (xu-cheng) [inline Library/Formula/prettyping.rb]:
```ruby
  homepage "https://denilsonsa.github.io/prettyping/"
  url "https://github.com/denilsonsa/prettyping/archive/v1.0.0.tar.gz"
  sha256 " SHA256 please "
```

**Comment by denilsonsa** [comment]
(Hopefully) fixed the stuff mentioned in the comments.

File: Library/Formula/prettyping.rb [hunk 9]
```diff
@@ -0,0 +1,13 @@
+class Prettyping < FileFormula
```
Comment (xu-cheng) [inline Library/Formula/prettyping.rb]: `FileFormula` -> `Formula`

File: Library/Formula/prettyping.rb [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+  homepage "http://denilsonsa.github.io/prettyping/"
```
Comment (xu-cheng) [inline Library/Formula/prettyping.rb]: Please use the `https` link.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `prettyping 1.0.0 (new formula)`.

**Pull request closed without merging**